.gitlab-ci.yml

```diff
@@ -1,7 +1,7 @@
 image: docker:latest
 
 services:
-    - docker:dind
+    - docker:19.03-dind
```

Comment:
@Ruesa18 best use `20.10-dind`. That's the same version we use in other pipelines

Reply:
@s4mpl3d we will take a look at this problem together tomorrow. Is that ok for you? 🙂